frontends/web/src/routes/account/summary/filters.tsx

```diff
@@ -23,12 +23,16 @@ export const Filters = ({
   display,
   disableFilters,
   disableWeeklyFilters,
+  showFiltersChartPercentageDiff,
   onDisplayWeek,
   onDisplayMonth,
   onDisplayYear,
   onDisplayAll
 }: TChartFiltersProps) => {
   const { t } = useTranslation();
+  if (!showFiltersChartPercentageDiff) {
+    return null;
+  }
```

Comment:
Since this property completely hides the whole component, is it really necessary? I think you could avoid creating it and just decide if including the filters at all in the parent component